ncm-ssh/src/main/perl/ssh.pm

```diff
@@ -47,6 +51,15 @@ sub valid_sshd_file
     return 1;
 }
 
+# escape values for the ssh config format.
+# adds \ before ?{}.()[]
+sub ssh_escape
```

Comment:
nvm, this used to be the case already, i see you just factored out the code below.